core/src/banking_stage/qos_service.rs

```diff
@@ -167,17 +168,20 @@ impl QosService {
         bank: &Bank,
     ) {
         let mut cost_tracker = bank.write_cost_tracker().unwrap();
+        let mut num_included = 0;
```

Comment:
maybe name this `num_tx_no_longer_in_flight`?

Reply:
I think I'm inclined to leave these as-is; `num_included` makes it match the variable naming here:
https://github.com/anza-xyz/agave/blob/182d27f718e1d6df347bdbe60edaba6026f1c6b5/core/src/banking_stage/qos_service.rs#L101

I think sticking with `num_included` helps reinforce that we're excluding tx's that were NOT added to the `CostTracker` initially